apps/examples/stylesheets/index.styl

```diff
@@ -0,0 +1,49 @@
+.Page--examples
+  &__cta
+    margin-left block_width_mar
+    margin-bottom 4em
+
+  &__example-section
+    margin 2em 0 6em
+
+  &__header
+    display flex
+    flex-direction row
+    margin-bottom 5em
+
+  &__title
+    width block_width_mar
+    > h3.Type
+      font-weight bold
+
+  &__description
+    max-width 40em
+
+  &__channels
+
+    .ChannelBlockGroup
+      margin-bottom 4em
+
+    .ChannelBlockGroup__header 
+      margin 2em .5em 1em
+
+    .ChannelBlockGroup__title > h4.Type
```

Comment:
Why not use the `&__` syntax with all of these? You also shouldn't be scoping blocks under other blocks like this.